client/src/reducers/colors.js

```diff
@@ -1,4 +1,4 @@
-import { createColors } from "../util/stateManager";
+import { ColorHelpers } from "../util/stateManager";
```

Comment:
Optional style suggestion, used in other reducers to make code cleaner: import with a short alias to make code below easier to read.

Ie, `import { ColorHelpers as CH } from ...`

and then `CH.createColors(...)`

See (for example) `reducers/categoricalSelection.sj`

Reply:
I'm going to actually not go with the abbreviation. I also couldn't find an example in the file you mentioned.  The comprehension cost outweighs the brevity imo